examples/LA/003_LA_ORU_R01_NBS_0_initial_obr_flatten_example_1.hl7

```diff
@@ -0,0 +1,76 @@
+MSH|^~\&|Natus^natus.health.state.la.us^DNS|Louisiana Office of Public Health Laboratory^2.16.840.1.114222.4.3.26.3.1^ISO|Test Department of tests^1.2.840.114350.1.13.286.2.7.2.695071^ISO|^simulated-hospital-id^L,M,N|20240101050000+0000||ORU^R01^ORU_R01|20240101050000_0001|T|2.5.1|||AL|AL|||||LRI_NG_FRN_PROFILE^^2.16.840.1.113883.9.195.3.4^ISO~LRI_NDBS_COMPONENT^^2.16.840.1.113883.9.195.3.6^ISO~LAB_PRN_Component^^2.16.840.1.113883.9.81^ISO~LAB_TO_COMPONENT^^2.16.840.1.113883.9.22^ISO|
```

Comment:
All sample files in the examples folder should have 'D' in MSH-11, with the exception of files examples/Test/Automated which should have 'N'. More info here: https://github.com/CDCgov/trusted-intermediary/tree/main/examples#sample-message-files

Reply:
Okay, I already have another branch that updates the file snapshots. I'll include this change in that branch instead so I have all file updates in one location. Is that okay?

Reply:
I made this change in the branch [update-snapshot-examples](https://github.com/CDCgov/trusted-intermediary/tree/updated-snapshot-examples)

Reply:
Could the files be removed from this branch and keep them in the other one to avoid confusion?

Reply:
K, I removed my added `examples/LA` files. They are all in the branch mentioned above.